src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs

```diff
@@ -31,16 +31,27 @@ public override ProcessingPriority GetProcessingPriority(FileAndType file)
             switch (file.Type)
             {
                 case DocumentType.Article:
-                    if (".yml".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase))
+                    if (".yml".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase) ||
+                        ".yaml".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase))
                     {
-                        return ProcessingPriority.Normal;
+                        var mime = YamlMime.ReadMime(Path.Combine(file.BaseDir, file.File));
```

Comment:
>ReadMime [](start = 44, length = 8)

I looked at ReadMime code, seems to be too simple. What if the first line is a normal yaml comment? Should a more strict format for yaml mime type (for example, must start with #yamlmime)?